src/main/resources/assets/vulkanmod/lang/ru_ru.json

```
@@ -0,0 +1,40 @@
+{
+  "vulkanmod.options.unknown": "Неизвестно",
+
+  "vulkanmod.options.pages.video": "Видео",
+  "vulkanmod.options.pages.graphics": "Графика",
+  "vulkanmod.options.pages.optimizations": "Оптимизация",
+  "vulkanmod.options.pages.other": "Другое",
+
+  "vulkanmod.options.buttons.apply": "Применить",
+  "vulkanmod.options.buttons.kofi": "Поддержите меня",
+
+  "vulkanmod.options.advCulling": "Отбраковка чанков",
+  "vulkanmod.options.advCulling.aggressive": "Агрессивная",
+  "vulkanmod.options.advCulling.conservative": "Сниженная",
+  "vulkanmod.options.advCulling.normal": "Обычная",
+  "vulkanmod.options.advCulling.tooltip": "Алгоритм отбраковкий, который может улучшить производительность благодаря уменьшению количества отображаемых чанков.",
+
+  "vulkanmod.options.ao.subBlock": "Вкл (Полублок)",
+  "vulkanmod.options.ao.subBlock.tooltip": "Вкл (Полублок): Мягкое освещение на неполных блоках (экспериментально).",
+
+  "vulkanmod.options.deviceSelector": "Выбор устройства",
+  "vulkanmod.options.deviceSelector.auto": "Авто",
+  "vulkanmod.options.deviceSelector.tooltip": "Текущее устройство",
+
+  "vulkanmod.options.entityCulling": "Отбраковка сущностей",
+  "vulkanmod.options.entityCulling.tooltip": "Отбраковывает сущностей в не видимых игроку областях.",
+
+  "vulkanmod.options.frameQueue": "Размер очереди рендеринга",
+  "vulkanmod.options.frameQueue.tooltip": "Более высокие значения могут помочь стабилизировать время между кадрами, но увеличат задержку ввода.",
+
+  "vulkanmod.options.indirectDraw": "Непрямая прорисовка",
+  "vulkanmod.options.indirectDraw.tooltip": "Уменьшает нагрузку на CPU, но увеличивает нагрузку на GPU. Включение может помочь в системах со слабым процессором.",
+
+  "vulkanmod.options.refreshRate": "Частота обновления",
+
+  "vulkanmod.options.uniqueOpaqueLayer": "Уникальный непрозрачный слой",
+  "vulkanmod.options.uniqueOpaqueLayer.tooltip": "Cлой рендеринга для непрозрачной местности, повышающий производительность.",
```

Вопрос: разве сам слой повышает производительность? Не то, что все opaque-текстуры выносятся на отдельный этап рендеринга?

У меня использовался более нейтральный вариант, может подойдёт:
```suggestion
  "vulkanmod.options.uniqueOpaqueLayer.tooltip": "Использовать отдельный слой рендеринга непрозрачной местности для повышения производительности.",
```